t/t6026-merge-attr.sh

```diff
@@ -180,4 +180,28 @@ test_expect_success 'up-to-date merge without common ancestor' '
 	)
 '
 
+cat >./custom-merge <<\EOF
+#$SHELL_PATH
+sleep 1 &
+EOF
+chmod +x ./custom-merge
+
+test_expect_success 'custom merge backend with delay' '
+	git reset --hard anchor &&
```

Comment:
This is just to make sure to get into a consistent state. (In case any of the previous tests have failed...)